README.md

```diff
@@ -1 +1,23 @@
-# startuphack
+# Gothenburg Startup Hack
+This is the official website for Gothenburg Startup Hack.
+
+## Setup
+
+### Prerequisites
+- Node.js (v7.0)
```

Comment:
up until v9 works. You tried with v10, right?

Reply:
Yes, did not work. We might have to consider looking over the build process and local server. It really is only for development, so it won't affect the site itself.